AndelaOSP/wire
# #52806729 Implement Social Login Logic

#### What does this PR do?
Implement Social Login Logic

#### Description of Task to be completed?
Implement Social Login Logic

#### Where should the reviewer start?
N/A

#### How should this be manually tested?
Start the project
Access http://wire.andela.com:8080/
You should see the login page.
Login with your Andelan Email
Upon successful login, you should see the admin dashboard

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[152806729](https://www.pivotaltracker.com/story/show/152806729)

#### Screenshots (if appropriate)
Login Screen
<img width="1280" alt="screen shot 2017-11-22 at 00 35 48" src="https://user-images.githubusercontent.com/25595450/33098058-229d6ca8-cf1d-11e7-9b52-ebad9f90d676.png">

On successful login
<img width="1280" alt="screen shot 2017-11-22 at 00 35 31" src="https://user-images.githubusercontent.com/25595450/33098067-2930673c-cf1d-11e7-9f10-d39a7e48c6bd.png">
